docker/ubuntu1604/Dockerfile

```diff
@@ -71,6 +75,15 @@ RUN update-alternatives --install /usr/bin/clang clang /usr/bin/clang-3.8 100 &&
     update-alternatives --install /usr/bin/scan-build scan-build /usr/bin/scan-build-3.8 100 && \
     update-alternatives --install /usr/bin/llvm-symbolizer llvm-symbolizer /usr/bin/llvm-symbolizer-3.8 100
 
+# Redis cannot start as a daemon on a Docker build.
+RUN sed -i "s/daemonize yes/daemonize no/g" /etc/redis/redis.conf
+
+# To preserve the logs for redis
+RUN chown -R redis:redis /var/log/redis/
```

Comment:
add to RUN above to avoid extra docker layer; these 2 are redis customisations that go together anyway
(even better to move those 2 steps into the whole install step above)